.github/workflows/tacos_unlock.yml

```diff
@@ -53,62 +64,50 @@ jobs:
         uses: ./tacos-gha/.github/actions/determine-slices
 
   tacos_unlock:
+    if: |
+      false
+      || github.event_name != 'label'
+      || github.event.label.name == ':taco::unlock'
+    needs: [determine-tf-root-modules]
     runs-on: ubuntu-latest
     permissions:
       contents: read
       pull-requests: write
       id-token: write
 
-    # note: we want to let people unlock even if they converted to draft first
-    if: |
-      github.event.label.name == ':taco::unlock'
-    outputs:
-      summary: ${{ steps.release-terraform-lock.outputs.summary }}
-    needs: [determine-tf-root-modules]
+    strategy:
+      fail-fast: false
+      matrix:
```

Comment:
I have no issues with the matrix, out of curiosity, what made you revert to it ?

Reply:
In short, that's the only thing that works today.
My attempt to allow for this non-matrix job was to create the with-user-env script, allowing `cd $TF_ROOT_MODULE` and direnv to be deferred much later in the process. This is a good idea and was intended to make TF_ROOT_MODULE be an optional variable, but the `setup` action depends on TF_ROOT_MODULE being set via the set-oidc-provider step. Rather than doubling down and taking time to "fix" it, here I've done the simplest thing and used what works today.

I may come back and finish this work, allow thins to run as a non-matrix job, but it seems very low priority at the moment.